src/civetweb.c

```diff
@@ -14192,7 +14192,10 @@ handle_request(struct mg_connection *conn)
 	                        &auth_callback_data,
 	                        NULL)) {
 		if (!auth_handler(conn, auth_callback_data)) {
-
+			mg_send_http_error(conn,
+			                   401,
+			                   "%s Unauthorized",
```

Comment:
@kica-z 
I guess it's okay to have a string `Unauthorized` here, but civetweb has a helper for that `mg_get_response_code_text`